Library/Formula/subversion.rb

```diff
@@ -138,7 +138,9 @@ def install
     if OS.mac? && MacOS::CLT.installed?
       args << "--with-apr=/usr"
       args << "--with-apr-util=/usr"
+      args << "--with-zlib=/usr"
     else
+      args << "--with-zlib=#{Formula["zlib"].opt_prefix}"
```

Comment:
This chunk isn't quite right for Mac, because zlib on Mac doesn't require that CLT be installed.
Please remove this chunk (both lines).